### PR TITLE
MID-2493: Update Gateway Docs

### PR DESCRIPTION
**Pull request by irishshagua**
* Clarify scope requirements after confusion arose during Hyperspace migration

Signed-off-by: bmooney <brian.mooney@zalando.ie>

### CODE REVIEW

**Comment by herojan** [comment]
Seems like there's too many changes here

File: docs/assets/basic-install.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  name: fabricgateways.zalando.org
+  name: "{{{ CRD_PLURAL }}}.zalando.org"
 spec:
   group: zalando.org
   version: v1
   names:
-    kind: FabricGateway
-    singular: fabricgateway
-    plural: fabricgateways
+    kind: "{{{ CRD_KIND }}}"
```
Comment (herojan) [inline docs/assets/basic-install.yaml]: This file was meant to have the kind etc hard-coded so people can do `kctl apply -f basic-install.yaml`

**Comment by herojan** [comment]
👍

**Comment by irishshagua** [comment]
👍

**Pull request merged**